classes/DepositPointsHelper.inc.php

```diff
@@ -23,16 +23,16 @@ class DepositPointsHelper {
 	 * @param $password string
 	 * @return array|null
 	 */
-	public static function loadCollectionsFromServer($url, $username, $password) {
-		if (empty($url) OR empty($username) OR empty($password)) {
-			return null;
-		}
+	public static function loadCollectionsFromServer($url, $username, $password, $apikey) {
 		$depositPoints = array();
-		$client = new SWORDAPPClient();
+		$client = new SWORDAPPClient([CURLOPT_HTTPHEADER => ["X-Ojs-Sword-Api-Token:".$apikey]]);
```

Comment:
Should this supply the API key if there is none? Off the top of my head, `$apiKey` should be an optional parameter, too.

Reply:
Fixed to make the param optional and only send it if it's present.